src/NuGet.Clients/NuGet.SolutionRestoreManager/BrokeredServicesUtility.cs

```diff
@@ -1,13 +1,24 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Threading.Tasks;
+using Microsoft.ServiceHub.Framework;
 using Microsoft.VisualStudio.Shell.ServiceBroker;
 
 namespace NuGet.SolutionRestoreManager
 {
     internal static class BrokeredServicesUtility
     {
+        // It is very important that these names and versions are kept the same as the ones in NuGet.VisualStudio.Internal.Contracts.NuGetServices.
+        internal static const string NuGetSolutionServiceName = "NuGetSolutionService";
+        internal static string NuGetSolutionServiceVersion = "1.0.0";
```

Comment:
Are these candidates for a constant to put in the Common project?

Reply:
They are only relevant for the service declaring assembly. 
And we do not want the contracts assembly to have any additional NuGet dependencies. 

So my thought is no, because no one else should care about these values.